migration/v0.20.x-v0.21.x/data-stream-rollovers.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -euo pipefail
+
+: "${CK8S_CONFIG_PATH:?Missing CK8S_CONFIG_PATH}"
+
+here="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+sops_config="${CK8S_CONFIG_PATH}/.sops.yaml"
+secrets="${CK8S_CONFIG_PATH}/secrets.yaml"
+
+OS_PASSWORD=$(sops --config "${sops_config}" -d --extract '["opensearch"]["adminPassword"]' "${secrets}")
+
+"${here}/../../bin/ck8s" ops kubectl sc exec -n opensearch-system opensearch-master-0 -c opensearch -- curl -XPOST 'http://localhost:9200/authlog/_rollover' -u "'admin:${OS_PASSWORD}'" --no-progress-meter
+echo
+"${here}/../../bin/ck8s" ops kubectl sc exec -n opensearch-system opensearch-master-0 -c opensearch --  curl -XPOST 'http://localhost:9200/kubeaudit/_rollover' -u "'admin:${OS_PASSWORD}'" --no-progress-meter
+echo
+"${here}/../../bin/ck8s" ops kubectl sc exec -n opensearch-system opensearch-master-0 -c opensearch --  curl -XPOST 'http://localhost:9200/kubernetes/_rollover' -u "'admin:${OS_PASSWORD}'" --no-progress-meter
```

Comment:
This would not really work for the customers where we have the "index per namespace" feature enabled. Don't know how we should handle that though :thinking:

Reply:
Maybe we could first list all aliases and then loop through them and do a rollover on each?

Reply:
Yep! Ended up doing that!